spec/03-plugins/10-key-auth/01-api_spec.lua

```diff
@@ -310,4 +311,101 @@ describe("Plugin: key-auth (API)", function()
       assert.equal(key_name, body.config.key_names[1])
     end)
   end)
+  describe("/key-auth", function()
+    local consumer2
+    describe("GET", function()
+      setup(function()
+        helpers.dao:truncate_table("keyauth_credentials")
+        for i = 1, 3 do
+          assert(helpers.dao.keyauth_credentials:insert {
+            consumer_id = consumer.id
+          })
+        end
+        consumer2 = assert(helpers.dao.consumers:insert {
+          username = "bob-the-buidler"
+        })
+        for i = 1, 3 do
+          assert(helpers.dao.keyauth_credentials:insert {
+            consumer_id = consumer2.id
+          })
+        end
+      end)
+      it("retrieve all the key-auths", function()
+        local res = assert(admin_client:send {
+          method = "GET",
+          path = "/key-auth/"
```

Comment:
We should test with, and without the trailing slash here is what I meant :) Sorry if it wasn't clear in my previous comment!

Reply:
@thibaultcha Sorry, my bad. I intended to do what you suggested but it somehow ended up being something else; fixed it.